doc/nrf/ug_tfm.rst

```diff
@@ -25,28 +25,36 @@ TF-M implementation in |NCS| is currently demonstrated in the :ref:`tfm_hello_wo
 Building
 ********
 
-To add TF-M to your build, enable the :option:`CONFIG_BUILD_WITH_TFM` config by adding it to your :file:`prj.conf` file.
-:note: If you instead enable :option:`CONFIG_BUILD_WITH_TFM` via menuconfig, you must also enable its dependencies.
-You must build with a nonsecure board. Currently these are supported:
+TF-M is one of the images that are built as part of a multi-image application.
+For more information about multi-image builds, see :ref:`ug_multi_image`.
 
- * nrf5340dk_nrf5340_cpuappns
- * nrf5340pdk_nrf5340_cpuappns
- * nrf9160dk_nrf9160ns
+To add TF-M to your build, enable the :option:`CONFIG_BUILD_WITH_TFM` configuration option by adding it to your :file:`prj.conf` file.
 
-If you are using nrf9160dk_nrf9160ns, copy the .overlay file from tfm_hello_world to disable UART1 in the nonsecure app, since this UART is used by the TF-M secure app.
+.. note::
+   If you use menuconfig to enable :option:`CONFIG_BUILD_WITH_TFM`, you must also enable its dependencies.
+
+TF-M must be built using a non-secure board target.
+The following boards are currently supported:
+
+* nrf5340dk_nrf5340_cpuappns
+* nrf9160dk_nrf9160ns
+
+If you are using nrf9160dk_nrf9160ns, copy the .overlay file from the :ref:`tfm_hello_world` sample to disable UART1 in the non-secure application, since this UART is used by the TF-M secure application.
```

Comment:
Note that the application will fail to run if this is not done on nrf9160dk_nrf9160ns.

Reply:
done